src/objects.js

```diff
@@ -1,41 +1,74 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name: name,
```

Comment:
This is good, when the name of your property is the same as the variable you want to assign to the value, you can write both at the same time. EG `{ name, age }` <= this is shorthand for `{ name: name, age: age }`. The only time you need to write both out is when the name of the variable you are assigning to the value is different to the name of the property eg `{ firstName: name, ageInYears: age }`